src/platform/testing/unit/mocha-setup.js

```diff
@@ -20,6 +20,12 @@ global.__MEGAMENU_CONFIG__ = null;
 chai.use(chaiAsPromised);
 chai.use(chaiDOM);
 
+function filterStackTrace(trace) {
+  return trace
+    .split('\n')
```

Comment:
Need to convert these to OS specific line breaks

Reply:
I think using `.split(/\r?\n/)` should work.

Reply:
That should work, or Node also has an `EOL` constant in the `os` module (https://nodejs.org/api/os.html#os_os_eol). 

I don't know if one option is better than the other.